cmd/cert-checker/main_test.go

```diff
@@ -29,8 +29,8 @@ import (
 )
 
 var (
-	saDbConnStr = "mysql+tcp://boulder@localhost:3306/boulder_sa_test"
-	paDbConnStr = "mysql+tcp://boulder@localhost:3306/boulder_policy_test"
+	saDbConnStr = "mysql+tcp://boulder@mysql:3306/boulder_sa_test"
```

Comment:
I think this will make tests break in the non-Docker world where `mysql` doesn't resolve. Maybe we can have this set by an env var, defaulting to `localhost` but overriden by Docker?